manifests/qpid.pp

```diff
@@ -0,0 +1,12 @@
+# Qpid setup for Candlepin
+class candlepin::qpid {
+
+  qpid::config::exchange { 'event':
```

Comment:
I believe this should be under an ```if $::candlepin::enable_amq``` somewhere. Either in this class or in init.pp.